src/features/metadata-instance-editor/Instance.js

```diff
@@ -592,6 +600,9 @@ class Instance extends React.PureComponent<Props, State> {
         }: State = this.state;
         const isProperties = template.templateKey === TEMPLATE_CUSTOM_PROPERTIES;
         const isEditing = this.isEditing();
+        const hasCascadePolicy = !!cascadePolicy?.id;
+        const hasCascadePermission = !hasCascadePolicy || (hasCascadePolicy && isCascadingPolicyApplicable);
+        const canEditInstance = !!(isEditing && hasCascadePermission);
```

Comment:
nit: `isEditing` and `hasCascadePermission` are booleans so you don't need the double bang
```suggestion
        const canEditInstance = isEditing && hasCascadePermission;
```

Reply:
isEditing can be undefined which is why I did this. Really, I should ensure that it's a boolean above

Reply:
`isEditing` being `undefined` sounds like a bug. do you know how that's happening? the methods should return booleans: https://github.com/box/box-ui-elements/blob/63d33f7e73f3cd53ebeaa7ae9804d02c5b9fb8f1/src/features/metadata-instance-editor/Instance.js#L534-L552

I was thinking with the `canEdit` prop but that should be required